src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+    .. Click _Import from source control_
++
+image::servicenow-import-from-source-control.png[align="center"]
+    .. Click _Import_. When the application import is completed, click the _Select Application button_.
++
+image::servicenow-imported-from-source-control.png[align="center"]
+    .. In the Select Application dialog, click the link to the new application to open it for editing in Studio.
+
+Because you will very likely customize the application, it's a good idea to create a new branch an start working from there
+
+[start=2]
+. [.underline]#Create a new branch#
+.. In Studio, open the _Source Control_ menu and select the _Create Branch_ menu item.
++
+image::servicenow-create-branch.png[align="center"]
+.. Configure the branch.
+
+    Branch Name: branchName
+    Create from Tag: -- None --
+
+.. Click the _Create Branch_ button.
+.. Click the _Close_ button.
+
+To load the application files included in the tag, return to the main ServiceNow browser tab (not Studio) and click the browser's reload button to refresh the page.
+
+===== Upload an Update Set
+
+Though not recommended by ServiceNow, you can download the following Update Set and import it in a ServiceNow instance
+to create an application. An Update Set describes and contains all application's files and is specific to ServiceNow.
+
+. link:../admin/references/servicenow/update_set_21813bcd2f9c91103c5d9facf699b605.xml[Download the Update Set].
+. Go to _Retrieved Update Sets_ under _System Update Sets_
+. Click on _Import Update Set from XML_
+. Select the Update Set you just downloaded
+. Click on _Upload_ and wait for the upload to complete
+
+The Retrieved Update Set table should be displayed with the freshly uploaded file named _ProActive Workflows & Scheduling_
+
+[start=6]
+. Click on the Update Set Name to open it.
+. Click on _Preview Update Set_ and wait for the preview to finish. +
+This step is important as it compares an update set retrieved from a remote instance to
+updates on the local instance to detect potential conflicts. You must preview an update set and address all problems before you can commit the update set.
+. Once finished, click on _Close_
+. Problems will appear because some tables and records don't exist in your instance and ServiceNow doesn't like that.
+That's because we are installing an application and not updating it. We can ignore them and commit the changes.
+.. Select all errors by clicking on the top left checkbox (this selects all the displayed errors)
+.. Click on _Accept remote changes_ on the top right
++
+image::servicenow-accept-remote-update.png[align="center"]
+.. Repeat step a. and b. if errors remain
+. Once all problems have been addressed. Click _Commit Update Set_.
+. Click Confirm data loss and wait for the commit to finish
+. Once again, and for the same reason ServiceNow will inform us of errors.
+.. Click on _Show Commit Log_ and check that all records are of "Info" status.
```

Comment:
There are three consecutive points. One after "errors" from the previous line and 2 before "Click" from the current line. I think you should keep only one.